<ui version="4.0" >
 <class>ExportPptDialog_base</class>
 <widget class="QDialog" name="ExportPptDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export HTML report to Powerpoint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelFileName" >
       <property name="text" >
        <string>HTML file name:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSelect" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Select the HTML file to export</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonProperties" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>HTML properties</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editFileName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupPptOptions" >
     <property name="title" >
      <string>Powerpoint options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBoxShowPptApplication" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Show Powerpoint application</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkBoxMinimizePptApplication" >
        <property name="text" >
         <string>Minimize Powerpoint application</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <widget class="QComboBox" name="comboPaperSize" >
          <item>
           <property name="text" >
            <string>A4</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Letter</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="labelPaperOrientation" >
          <property name="text" >
           <string>Paper orientation:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>comboPaperOrientation</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="labelPaperSize" >
          <property name="text" >
           <string>Paper size:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>comboPaperSize</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboPaperOrientation" >
          <item>
           <property name="text" >
            <string>Landscape</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Portrait</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>16</width>
       <height>85</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonExport" >
         <property name="text" >
          <string>&amp;Export</string>
         </property>
         <property name="shortcut" >
          <string>Alt+E</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
         <property name="shortcut" >
          <string>Alt+C</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
